packages/boot/tools/supports.ts

```diff
@@ -362,11 +363,41 @@ export const makeSwingsetTestKit = async (
           case 'IBC_METHOD':
             switch (obj.method) {
               case 'startChannelOpenInit':
-                inbound(
-                  BridgeId.DIBC,
-                  icaMocks.startChannelOpenInit.channelOpenAck(obj),
-                );
+                inbound(BridgeId.DIBC, icaMocks.channelOpenAck(obj));
                 return undefined;
+              case 'sendPacket':
+                switch (obj.packet.data) {
+                  case protoMsgMocks.delegate.msg: {
+                    ibcSequenceNonce += 1;
+                    const msg = icaMocks.ackPacket(
+                      obj,
+                      ibcSequenceNonce,
+                      protoMsgMocks.delegate.ack,
+                    );
+                    inbound(BridgeId.DIBC, msg);
+                    return msg.packet;
```

Comment:
this is mostly repeated three times. please DRY. e.g.
```suggestion
                    return icaMocks.ackPacket(
                      protoMsgMocks.delegate.ack,
                      obj,
                      ibcSequenceNonce,
                      inbound,
                    );
```

Reply:
There are two things happening here - a call to `inbound` and a return. Both are necessary, so I created an intermediate variable for some DRYness. Factored out a bit more to `makeAckEvent`.

Reply:
I had in mind that the called function returning a value would do the call to `inbound`. That's why I included it as a param in the function call. But it's not worth another CI rev